### DISCUSSION TITLE
Ddt4all reset go wrong R link2 2018 Megane

### DISCUSSION THREAD

**Discussion by Endkod** [cedricp/ddt4all#1258] [Q&A]
Can someone help me with this problem.
No sound/camera/temperature control after ddt4all reset. The screen was also black, which I resolved with a macro, which can still be done to get it working again.

**Comment by AlMeu** [comment]
> Can someone help me with this problem. No sound/camera/temperature control after ddt4all reset. The screen was also black, which I resolved with a macro, which can still be done to get it working again.

What software version you have in your unit?
A photo will help to understand your situation.

**Comment by Endkod** [comment]
![IMG_20230707_213253503](https://github.com/user-attachments/assets/2bd570c0-4045-41c2-8c8d-b2b37398e496)
Before

**Comment by AlMeu** [comment]
The upgrade from v2.2 or v3.3 to higher version is done only in Renault dealer services or by someone who knows very well what and how to do.
I messed up the software upgrade wrong.
Go to a Renault service maybe they will get fixed.

**Comment by Endkod** [comment]
I think I reset the radio dtc because I had a fault in my tire sensor. Renault quickly checked with the can clip but could not update the radio, that was on my holiday in France. They said error in radio and automatic transmission. Megane DCI gt was delivered with software 7. and is from 2018. I think I need a macro to set everything right.

**Comment by Endkod** [comment]
R link 2 screen works just shows radio time... but no sound, no multi sense, camera

**Comment by Endkod** [comment]
I have tried to perform an radio update myself with Canclip but it stops at 55% error automatic gearbox please contact information......

**Comment by AlMeu** [comment]
Following your unit behavior as you describe it is looking like an upgrade from v3.3 to v9.0 wrong. These are not settings problem.
If the CanClip refuse the upgrade it is end of story.
Just remains to change the unit with another new unlocked.

**Comment by Endkod** [comment]
Then I made a mistake with Ddt4all, I reset the radio to DTC. R link software 9. was upgraded last year from 7 to 9. It just worked until I reset it with Ddt4all 2 weeks ago

**Comment by matzel82** [comment]
Disconnect Clip and Battery and wait a time at 1h.
Than reconnect. Try Update again.

**Comment by AlMeu** [comment]
If you say so, that it is.
But the fact it is software malfunction in your unit and not settings wrong

**Comment by Endkod** [comment]
![Schermafbeelding 2024-09-24 110906](https://github.com/user-attachments/assets/02fcf26a-b947-48e7-bc50-0e3172ddf571)
This is how the screen now looks very standard. All other controls are not present.  Park Assist Display on cockpit does work, which I had set last year

**Comment by Endkod** [comment]
The temperature control is not visible, but the buttons do work when you turn it for hot or cold. And no camera, sound, multi sense, etc.

**Comment by Endkod** [comment]
USB also works

**Comment by jinzo** [comment]
And you did try to 'restore' factory configuration with the data the Telegram pyren bot gives you?

**Comment by Endkod** [comment]
How to do that

**Reply by jinzo** [reply]
https://github.com/cedricp/ddt4all/issues/784 More info with an example

**Comment by Endkod** [comment]
Now my R link does nothing at all, black screen and usb does nothing either, so it doesn't charge anymore. Tried downgrade but the R link is completely dead now

**Reply by AlMeu** [reply]
If apart black screen you have followings:
- no sound in speakers
- dashboard flickering in 2 colors
- USB not working
your unit is dead.
I can help you with another unit if you leave here an email for contact you.

**Reply by Endkod** [reply]
What does another Unit cost

**Reply by Endkod** [reply]
I have an appointment with Renault next week, I want to see what they say first. I will then contact you, but I would like to know in advance what another radio unit costs
